### PR TITLE
course: task 4.2 counter

### PR DESCRIPTION
**Pull request by BorislavAnchev**
The counter task with some improvements, working imports
and working Karma Runner.

### CODE REVIEW

**Comment by mgenov** [comment]
node_modules should not be committed as package.json and package-lock.json are available in the repository.

**Comment by BorislavAnchev** [comment]
The pull request should be updated. I don't know if it notifies you when I make changes or only when I create the PR initially. Should I write comments here when the 'fixes' are ready for another review or should I just git push it?

File: task42/karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    // Concurrency level
+    // how many browser should be started simultaneous
+    concurrency: Infinity
+  })
+}
+
+// OTHER CONfig
+
+// module.exports = function (config) {
```
Comment (mgenov) [inline task42/karma.conf.js]: Commented code should be removed.

**Comment by mgenov** [comment]
You have to write a message to let the reviewer knows that something was updated.

**Comment by BorislavAnchev** [comment]
Missed old config is removed.

File: task42/tests/main.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { counter } from '../src/counter';
+
+describe('#counter', function () {
+  it('should increment current value by 1 and return 1', function () {
```
Comment (mgenov) [inline task42/tests/main.spec.js]: "0 increment with 1 is 1"

sounds more reasonable as "current value" is value in the test and the developer has to read it to know.

**Comment by BorislavAnchev** [comment]
Fixes are made.

**Pull request merged**